docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: '3.12'
+
+services:
+  postgres:
+    image: postgres:16
```

I think it will be better to use the latest tag, so we don't need to update this manually in the future.
```suggestion
    image: postgres
```

---

I don't agree. What if new postgres version comes up and Django doesn't support it yet?

---

I don't think there is high chance of this, and I prefer to be sure I use the latest release, but this is fine too.